src/app/assets/Icons/PrimPrioIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export default function PrimPrioIcon(): JSX.Element {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="50"
+      height="50"
+      viewBox="0 0 50 50"
+    >
+      <g transform="translate(-811 120)">
+        <g
+          transform="translate(811 -120)"
```

Comment:
These `g` reverse each other.
You can remove the outer `g` and `transform`.

Reply:
Done. Thank you.